bin/cron/csf_workshop_attendance_to_gdrive

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env ruby
+require_relative 'only_one'
+abort 'Script already running' unless only_one_running?(__FILE__)
+
+require_relative '../../dashboard/config/environment'
+require 'cdo/google/sheet'
+
+#
+# Uses a Google Cloud service account to write enrollment data for CSF workshops into
+# a spreadsheet in Google Drive (with permissions locked down to our organization).
+#
+
+EARLIEST_WORKSHOP_END_DATE = Date.new(2020, 6, 1)
+LATEST_WORKSHOP_START_DATE = Date.today
+
+def get_rows
+  # Seed results array with a header row
+  results = [
+    [
+      "Organizer Name",
+      "Organizer Email",
+      "Regional Partner Name",
+      "Workshop Dates",
+      "Funded",
+      "Attendance Url",
+      "Num Facilitators",
+      "Course",
+      "Subject",
+      "Is this a Virtual Workshop?",
+      "Total Hours",
+      "Num Registered",
+      "Num Scholarship Teachers Attending Day 1",
+      "Attendance Day 1"
+    ]
+  ]
+
+  Pd::Workshop.
+    where(subject: Pd::SharedWorkshopConstants::SUBJECT_CSF_101).
+    scheduled_end_on_or_after(EARLIEST_WORKSHOP_END_DATE).
+    scheduled_start_on_or_before(LATEST_WORKSHOP_START_DATE).find_each do |workshop|
+      results << [
+        workshop.organizer&.name,
+        workshop.organizer&.email,
+        workshop.regional_partner&.name,
+        workshop.sessions.map(&:formatted_date_with_start_and_end_times).join("\n"),
```

Comment:
We want new lines in the gsheet export?

Reply:
It creates a list of the workshop sessions:
<img width="322" alt="Screen Shot 2021-02-21 at 1 04 15 PM" src="https://user-images.githubusercontent.com/224089/108638890-56ef7b00-7446-11eb-8530-ace74705f511.png">